code_comment/lib.py

```diff
@@ -116,13 +170,14 @@ def is_single_line_comment_multiline_notation(text):
                 not is_currently_multi_line_comment()
                 and text.startswith(mlc_header)
                 and text.endswith(mlc_footer)
+                and len(text) >= 6
             )
 
         def is_multi_line_comment_start(text):
             return (
                 not is_currently_multi_line_comment()
                 and text.startswith(mlc_header)
-                and not text.endswith(mlc_footer)
+                and (not text.endswith(mlc_footer) or len(text) == 3)
```

Comment:
same as suggested above.

```suggestion
                and (not text.endswith(mlc_footer) or len(text) == len(mlc_footer)
```